src/starter/files-white-label/src/app/core/services/theming/theming.service.ts

```diff
@@ -1,34 +1,34 @@
-import { Injectable, ElementRef } from '@angular/core';
+import { DOCUMENT } from '@angular/common';
+import { Injectable, Inject } from '@angular/core';
 import { AppConfiguration } from '../configuration/configuration';
 
 @Injectable({
   providedIn: 'root',
 })
 export class ThemingService {
-  setCSSVariables(el: ElementRef, propertyMap: AppConfiguration): void {
-    el.nativeElement.style.setProperty(
+  constructor(@Inject(DOCUMENT) private document: Document) {}
```

Comment:
a ThemingService precisa de um construtor com o Document injetado? usa-se o `this.document` em algum momento?

Reply:
Verdade. Não precisa.